langchain4j-agentic/src/test/java/dev/langchain4j/agentic/Agents.java

```diff
@@ -24,7 +25,8 @@ public interface ExpertRouterAgentWithMemory extends AgenticScopeAccess {
 
     public interface CategoryRouter {
 
-        @UserMessage("""
+        @UserMessage(
```

Comment:
Can you please rollback and avoid all the useless code reformatting in this class?

Reply:
These changes are generated by ` ./mvnw spotless:apply`